subiquitycore/models/network.py

```diff
@@ -23,6 +23,7 @@
 
 NETDEV_IGNORED_IFACE_NAMES = ['lo']
 NETDEV_IGNORED_IFACE_TYPES = ['bridge', 'tun', 'tap', 'dummy', 'sit']
+NETDEV_WHITELIST_IFACE_TYPES = ['vlan']
```

Comment:
I think the name of this list should have VIRTUAL in it somewhere?

Reply:
https://trello.com/c/8FdcEE3N